docs/user-guide/doc-odm-user-guide/setup.md

```diff
@@ -1,81 +1,229 @@
+
+Only Administrators can add or remove members to the Curator Group. To Add new members, simply follow the instructions described in previous sections and select the User to add.
+
+1. Briefly, navigate to the Groups section. Select the group Curator
+2. Click on **+New member** and select the member to add
+3. Confirm the changes
+4. Once the user is added, the title will be displayed on its profile. 
+
+![29Curator.png](doc-odm-user-guide/images/29Curator.png)
+<figcaption>To add a new member to the Curator group, navigate to the Groups page, select the Curator group and click on <strong>+New member</strong>. Select the user to add and save the changes. The new user e.g. <strong>Public</strong>, will be now added to the Curator group and the information will be displayed on the user profile</figcaption>
+
+Curators can explore, edit, and curate data. However, some limitations are set based on their permissions:
+
+* **Curators with Manage Groups permissions**: Members of the Curator group with full edit permissions. Users can add or remove members of the Curator Group
+
+![40Curator.png](doc-odm-user-guide/images/40Curator.png)
+<figcaption>Curators with Manage Group permissions can edit the Curator groups. Users can add and remove users, edit permissions and even delete groups</figcaption>
+
+* **Curators without Manage Groups permissions**: Curators without the permission to manage groups, can still explore and edit data. However, adding or removing users within groups (including the Curator Group) is unavailable
+
+![31CuratorLim.png](doc-odm-user-guide/images/31CuratorLim.png)
+<figcaption>Member of the curator groups but without Manage Groups permissions can still have access to data and curation options, however, users cannot add or remove members</figcaption>
+
+!!! danger
+    It is not advisable to delete the **Curator Group**, since Users need to be aware of potential changes
+
+## Sharing Data
+
+To share a study, you must be the study owner (the user who imported it to ODM). The share option is available in the Study menu. 
+
+![41Share.png](doc-odm-user-guide/images/41Share.png){width=600}
+<figcaption>To share a study, open the study of interest and click on the top bar menu. Select Share. A window will open with the available groups to share the study with</figcaption>
+
+**Only owners can share the study**. If a user is a member of the curator group, the user can still access the study and curate data, however, the study cannot be shared with other groups.
+
+![42ShareLim.png](doc-odm-user-guide/images/42ShareLim.png)
+<figcaption>Owners can share studies only with groups they are members of</figcaption>
+
+!!! info
+    If you are not an owner, you can browse the owner and the groups it is shared within **More info** and request the owner to share it. If the owner is unavailable, a new owner can be assigned using a script by an administrator.
```

Comment:
>If the owner is unavailable, a new owner can be assigned using a script by an administrator.

can we do not mention it? we don't have production-ready functionality to change study owner, the script that we had has been tested once and send to only one client long time ago, we don't know if it still works correctly.